src/test/keyword-mark_test.ts

```diff
@@ -26,6 +26,11 @@ describe('keyword-mark', () => {
     expect(element.innerHTML).to.equal('foo BAR baz');
   });
 
+  it('should do nothing if no keywords with custom delimiter', () => {
+    element.delimiter = ',';
+    expect(element.innerHTML).to.equal('foo BAR baz');
```

Comment:
this will always be true i think because we never mutate the light DOM, possibly should be asserting that the shadow DOM result is `foo BAR baz` (without any marks) instead?